docs/features/security/auth/jwt.md

```diff
@@ -62,4 +62,55 @@ public class Startup
         });
     }
 }
+```
+
+### Custom Claim validation
+
+It allows validating not only not claims that are added to the authorization in the JWT token, but any type of custom claim that needs to be verified
```

Comment:
```suggestion
It allows validating not only on the audience claim in the JWT token, but any type of custom claim that needs to be verified
```

Double negation. I guess you meant this :)

Reply:
done